docs/_sections/tables.html

```diff
@@ -384,19 +384,19 @@ <h3 id="pagination">Search Results Pagination</h3>
       <span class="nypl-example-text">Example</span>
 <div class="nypl-results-pagination">
   <a href="/page/1" rel="prev">
-    <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" viewBox="0 0 32 32">
-<title>Left arrow</title>
-<polygon points="16.959 24.065 9.905 16.963 27.298 16.963 27.298 14.548 9.905 14.548 16.959 7.397 15.026 5.417 4.688 15.707 15.026 25.998 16.959 24.065" />
+    <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" viewBox="0 0 8.97 15.125">
+      <title>Wedge Left Arrow</title>
+      <polygon points="7.563 15.125 0 7.562 7.563 0 8.97 1.407 2.815 7.562 8.97 13.717 7.563 15.125" />
     </svg>
-    Previous page
+    Previous
   </a>
   <span class="page-count">Page 2 of 404</span>
```

Comment:
I think `aria-live` would go on the next/prev links. I have to recheck what we did for new arrivals and get a refresher on that. Yes, it is relevant. It's not a new page refresh (unless if you turn JS off), so it'd be helpful since the results get updated.